tests/run-make/doctests-keep-binaries-2024/rmake.rs

```diff
@@ -16,7 +16,22 @@ fn setup_test_env<F: FnOnce(&Path, &Path)>(callback: F) {
 }
 
 fn check_generated_binaries() {
-    run("doctests/merged_doctest_2024/rust_out");
+    let mut found_merged_doctest = false;
+    rfs::read_dir_entries("doctests/", |path| {
```

Comment:
Ah right, now there is a hash in the generated file, not only the edition. Makes sense.

Reply:
Yeah, it's necessary otherwise we would write to the same directory.

I don't really like the hash, but it's better than appending the attributes to the filename, which would quickly it file path limit on many platform.